online-mpi/src/main/java/eu/itesla_project/online/mpi/Master.java

```diff
@@ -82,13 +82,13 @@ public static void main(String[] args) throws Exception {
 
             MpiExecutorContext mpiExecutorContext = new MultiStateNetworkAwareMpiExecutorContext();
             ScheduledExecutorService scheduledExecutorService = Executors.newScheduledThreadPool(1);
-            ExecutorService executorService = MultiStateNetworkAwareExecutors.newCachedThreadPool();
+            //ExecutorService executorService = MultiStateNetworkAwareExecutors.newCachedThreadPool();
```

Comment:
Why network aware executor is not used anymore? It multi-state IIDM network used inside the online process? network aware executor ensure current state in correctly set when getting data from a network in a new thread.